Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2_4_5.yml

```
@@ -1764,15 +1559,96 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "21":
+    id: "21"
+    taskid: 7d2bdab4-aee1-4922-87a4-19eb4d35e264
+    type: condition
+    task:
+      id: 7d2bdab4-aee1-4922-87a4-19eb4d35e264
+      version: -1
+      name: Is Image File?
```

Let's keep the convention as the other tasks. I think this phrasing reflects the logic better:
```suggestion
      name: Is there an image file?
```